.azure-pipelines/templates/test-all.yml

```diff
@@ -6,6 +6,7 @@ parameters:
   run_py2_tests: true
   force_base_package: false
   coverage: true
+  test_e2e: false
```

Comment:
If default is going to be false then you need to set it to true in changes.yml

Reply:
@hithwen Seems better to default to true then ?

Reply:
Yes, I think so

Reply:
Thanks, changed.